environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -124,6 +138,9 @@ resources:
             if [[ `echo $role |awk -F"-" '{print $2}'` == "contrailanalytics" ]]; then
               yum remove -y python-redis
             fi
+            if [[ `echo $role` != "overcloud-contrailcontroller-3" && `echo $role` != "overcloud-contrailcontroller-4" && `echo $role` != "overcloud-contrailcontroller-5" && `echo $role` != "overcloud-contrailanalytics-3" && `echo $role` != "overcloud-contrailanalytics-4" && `echo $role` != "overcloud-contrailanalytics-5" && `echo $role` != "overcloud-contrailanalyticsdatabase-3" && `echo $role` != "overcloud-contrailanalyticsdatabase-4" && `echo $role` != "overcloud-contrailanalyticsdatabase-5"]]; then
```

Comment:
1) It is better to check this condition before to create new repo.
if [[ ${contrail_repo_new} ]] ; then
   if [[ ... ]] ; then
   fi
fi

2) you don not check for compute, contraildpdk, contrailtsn - is it correct?

3) check of names should be node as in other places by 'short' part of the name, see checking for novacompute for example. (and I would avoid many 'awk' there by having it one to get short_role, e.g. short_role=$(echo $role |awk -F"-" '{print $2}'),   and then just use "$short_role" != '..')

Reply:
1. Already checking here: https://github.com/Juniper/contrail-tripleo-heat-templates/pull/42#discussion-diff-148210703R107

2. Correct: I've tested only for KVM compute case.

3. Good point. We can include this change at a later point in time.